content/_code-samples/submit-and-verify/py/submit-and-verify.py

```diff
@@ -0,0 +1,38 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import AccountSet
+from xrpl.transaction import safe_sign_and_submit_transaction
+from xrpl.core import keypairs
+from xrpl.wallet import Wallet
+
+myAddr = "rfaNymVS1fEREj4FpNroXVEcVVMph6k2Mt"
+mySeed = "ss3GZzmyEBxvwB3LUv-----------"
+
+# Derive and initialize wallet
+public, private = keypairs.derive_keypair(mySeed)
+wallet_from_seed = Wallet(mySeed, 0)
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Construct AccountSet transaction
+tx = AccountSet(
+    account=myAddr,
+    fee="10",
+    set_flag=1  # Numerical Value: 1 = asfRequireDest
```

Comment:
```suggestion
```
To specify a flag here could be confusing for the user. The [example](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/submit-and-verify) also didn't specify any flags.

Reply:
You're right, flags can be a confusing concept to new devs early on.